src/test/java/hudson/tasks/AntTest.java

```diff
@@ -123,6 +124,12 @@ private void verify() throws Exception {
         assertEquals(1,l.length);
         r.assertEqualBeans(l[0],new AntInstallation("myAnt","/tmp/foo", JenkinsRule.NO_PROPERTIES),"name,home");
 
+        // Verify that PATH+ANT is set.
```

Comment:
Of course core unit-tests that behavior, but this would be proving that the purported fix actually has some effect: that the method you are overriding is called when you think it is, etc.